build-info-extractor-gradle/src/test/java/org/jfrog/gradle/plugin/artifactory/GradlePluginTest.java

```diff
@@ -110,11 +110,29 @@ public void ciServerTest(String gradleVersion) throws IOException {
         cleanTestBuilds(buildDetails.getLeft(), buildDetails.getRight(), null);
     }
 
+    @Test(dataProvider = "gradleVersions")
+    public void deprecatedCiServerTest(String gradleVersion) throws IOException {
```

Comment:
Let;s add a description, explaining what's tested by this function. The random reader will not know what's deprecated, why and what this test is all about.